README.md

```diff
@@ -238,14 +239,14 @@ splunk_conf 'system/test.conf' do
 end
 ```
 
-###### Stanza-level Evaluation
+###### Section-level Evaluation
 
-Stanza-level evaluation allows you to evaluate the config key values for a specific stanza without being
+section-level evaluation allows you to evaluate the config key values for a specific section without being
 responsible for regurgitating all of the config you didn't want to modify.
 
-In this example, only a specific stanza's contents are turned into lowercase strings. The rest is
-merged as usual. **Note that the Stanza-level proc receives the section string and a props hash, and expects
-a pair (array) of the desired section string and props hash.**
+In this example, only a specific section's contents are turned into lowercase strings. The rest is
+merged as usual. **Note that the section-level proc receives the context and a props hash, and expects
+a pair (array) of the desired section name and props hash to be returned.**
```

Comment:
Yes, you're right. It made sense at the time, when stanza name and contents were being provided, to expect those to be returned. Since we have the context now, and every level of available evaluation, we don't need that.